test/integration/host_test.rb

```diff
@@ -342,7 +339,7 @@ def test_show_action_buttons
       page.find('#check_all').click
 
       # Ensure all hosts are checked
-      assert page.find('input.host_select_boxes').checked?
+      assert page.all('input.host_select_boxes').map(&:checked?).uniq
```

Comment:
With transactional fixtures, there are going to be more than just one host, so 'check all' will check more than 1 box.